src/events/events.ts

```diff
@@ -110,6 +110,51 @@ export function decodeLzPacketEvent(
     }
   }
 
+  // NOTE: This function can be optimized as its currently inefficient
+
+  // This is a fallback for the legacy LayerZeroModule contract address that was updated for all networks
+  const toFind3 = '0x6f484Eacd997D9880205aF22f6a4881ea0e1CCd7'.toLowerCase().slice(2, 42)
```

Comment:
maybe instead of just numbering these toFind functions it would be nicer to name thing something more descriptive like `toFindFallbackLZ` or something like that